packages/code-sandbox/src/callbackLogFormatter.ts

```diff
@@ -0,0 +1,20 @@
+import { formatCode } from '@stardust-ui/docs-components'
+import * as React from 'react'
+
+const callbackLogFormatter = (props: string[]) => (
```

Comment:
Can we have some documentation on how to use `callbackLogFormatter`?  It's not clear at first glance, what the parameter is about

Reply:
I added a comment there.